Makefile

```diff
@@ -9,16 +10,17 @@
 # Y - in response to Would you like to add the Hotwire addon?
 
 create_web:
-	printf "Y\nY\nY\nY\nY\nY\nY\nY\nY\n" | rails new $(APP_NAME) -m ./template.rb -T ${OPTIONS}
+	printf "Y\nY\nY\nY\nY\nY\nY\nY\nY\nY\n" | rails new $(APP_NAME) -m ./template.rb -T ${OPTIONS}
 
 # Y - in response to Would you like to add the Github addon?
+# Y - in response to Would you like to add the Mock Server addon?
 # Y - in response to Would you like to add the SemaphoreCI addon?
```

Comment:
```suggestion
# Y - in response to Would you like to add the SemaphoreCI addon?
# Y - in response to Would you like to add the Mock Server addon?
```

Reply:
Sorry, my eyes where looking at the web part, 


<img width="231" alt="image" src="https://github.com/nimblehq/rails-templates/assets/77609814/7970dfcc-8c0c-496b-852c-1608d164fc6a">

so when I saw 

<img width="157" alt="image" src="https://github.com/nimblehq/rails-templates/assets/77609814/efaa0038-4af6-446d-a2a7-2c861d67cecb">

I thought the ordering was wrong :harold: